website/other/bezier-rs-demos/src/App.vue

```diff
@@ -558,6 +566,38 @@ export default defineComponent({
 					name: "Length",
 					callback: (subpath: WasmSubpathInstance): string => subpath.length(),
 				},
+				{
+					name: "Evaluate",
+					callback: (subpath: WasmSubpathInstance, options: Record<string, number>): string => subpath.evaluate(options.t),
+					sliderOptions: [tSliderOptions], // TODO: add slider to toggle between parametric and euclidean
```

Comment:
A few things:
- This should not be called a slider, call it either a checkbox or toggle. (A slider means a [range input](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range))
- But when I merged Hannah's parametric/euclidean PR, I replaced that checkbox/toggle input with a radio button, please see how it looks now in the Evaluate function at <https://graphite.rs/bezier-rs-demos/>
- If the intention (as I'm assuming based on glancing at this code) is to separately implement the parametric/euclidean toggle, please directly copy the pertinent code from those Vue components now that I changed it to a pair of radio buttons.
    - The goal in the near future is to combine `BezierExample.vue` and `SubpathExample.vue`, as well as `BezierExamplePane.vue` and `SubpathExamplePane.vue`, into one component since that is duplicated code that makes maintenance (like this kind of dual implementation) more burdensome. I hope you guys can aim to put that goal on the roadmap for the relatively near future.
    - That'll be the first step towards removing Vue entirely. The next step will be merging `*ExamplePane.vue` into `App.vue`, and then `*Example.vue` also into `App.vue` so that becomes the only Vue component, prior to fully removing Vue.

So in summary: please remove all the code related to "slider options" (since that's the wrong terminology, but also because you'll want to directly copy the radio button-based implementation). Thanks! Also if you can get this done before Monday night, I will be able to review it before I leave early vacation Tuesday morning.